modules/apps/layout/layout-service/src/main/java/com/liferay/layout/internal/upgrade/registry/LayoutServiceUpgradeStepRegistrator.java

```diff
@@ -130,6 +138,10 @@ public void register(Registry registry) {
 	@Reference
 	private LayoutBranchLocalService _layoutBranchLocalService;
 
+	@Reference
+	private LayoutClassedModelUsageLocalService
```

Comment:
@gergelyszalay we cannot use services from the same module here, so we have to switch to SQL